index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
Not bad, for browser games this works well, though if we do port it to PC or other devices like consoles, maybe setting this to a button and changing the text would be good
for now its good!